src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -32,6 +32,11 @@
 #include <uct/ib/ud/base/ud_inl.h>
 
 
+#define uct_ud_mlx5_wqe_av_size(_av) \
```

Comment:
could just make it part of uct_ud_mlx5_ep_ctrl_av_size

Reply:
> could just make it part of uct_ud_mlx5_ep_ctrl_av_size

ok, inlined